tests/components/bryant_evolution/test_config_flow.py

```diff
@@ -0,0 +1,171 @@
+"""Test the Bryant Evolution config flow."""
+
+from unittest.mock import DEFAULT, AsyncMock, patch
+
+from evolutionhttp import BryantEvolutionLocalClient, ZoneInfo
+
+from homeassistant import config_entries
+from homeassistant.components.bryant_evolution.const import CONF_SYSTEM_ZONE, DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import CONF_FILENAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+
+async def test_form_success(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+    assert result["type"] is FlowResultType.FORM
+    assert result["errors"] == {}
+
+    with (
+        patch.object(
+            BryantEvolutionLocalClient,
+            "enumerate_zones",
+            return_value=DEFAULT,
+        ) as mock_call,
+    ):
+        mock_call.side_effect = lambda system_id, filename: {
+            1: [ZoneInfo(1, 1, "S1Z1"), ZoneInfo(1, 2, "S1Z2")],
+            2: [ZoneInfo(2, 3, "S2Z2"), ZoneInfo(2, 4, "S2Z3")],
+        }.get(system_id, [])
```

Comment:
Can we use the same mock as we have for the climate tests?

Reply:
this is testing a different path -- those are mocking get_client; this is mocking enumeate_zones.